packages/dialtone-vue2/recipes/conversation_view/message_input/message_input.vue

```diff
@@ -170,12 +170,12 @@
 
         <!-- Send button -->
         <dt-tooltip
-          v-if="showSend"
+          v-if="true"
```

Comment:
If this is always true, isn't it better to remove it?

Reply:
agh thanks for noticing that, that was for testing only, will change back.

Reply:
fixed.